packages/core/src/components/SwipeableItem/SwipeableItem.tsx

```diff
@@ -124,9 +125,18 @@ const SwipeableItem: React.FC<React.PropsWithChildren<Props>> = ({
   const [componentWidth, setComponentWidth] = React.useState<number | null>(
     null
   );
+
+  const children: React.ReactNode[] = React.useMemo(
```

Comment:
Can we get a comment explaining why we need to pull extract components out of fragments in the children prop?